services/web/server/tests/unit/with_dbs/06/test_projects_02.py

```diff
@@ -626,40 +628,56 @@ async def test_close_project(
     user_project,
     client_session_id_factory: Callable,
     expected,
-    mocked_director_subsystem,
+    mocked_director_api,
     fake_services,
 ):
     # POST /v0/projects/{project_id}:close
     fakes = fake_services(5)
     assert len(fakes) == 5
-    mocked_director_subsystem[
-        "get_running_interactive_services"
-    ].return_value = future_with_result(fakes)
+    mocked_director_api["get_running_interactive_services"].return_value = fakes
 
     # open project
     client_id = client_session_id_factory()
     url = client.app.router["open_project"].url_for(project_id=user_project["uuid"])
     resp = await client.post(url, json=client_id)
 
     if resp.status == web.HTTPOk.status_code:
-        calls = [
-            call(client.server.app, user_project["uuid"], logged_user["id"]),
-        ]
-        mocked_director_subsystem["get_running_interactive_services"].has_calls(calls)
-        mocked_director_subsystem["get_running_interactive_services"].reset_mock()
+        mocked_director_api["get_running_interactive_services"].assert_any_call(
+            client.server.app, logged_user["id"], user_project["uuid"]
+        )
+        mocked_director_api["get_running_interactive_services"].reset_mock()
```

Comment:
I think here you should use ```assert_called_once_with``` what you did here changes the result of the test

Reply:
I preserved the previous asserts. I would prefer to improve the test in a subsequent PR

Reply:
not sure I understand the reason here. before it was looking for only one call. now you assert for any number.